packages/liveblocks-core/src/index.ts

```diff
@@ -35,7 +35,8 @@ export type {
   LiveObjectUpdate,
   StorageUpdate,
 } from "./crdts/StorageUpdates";
-export type { ToImmutable, toPlainLson } from "./crdts/utils";
+export type { ToImmutable } from "./crdts/utils";
+export { toPlainLson } from "./crdts/utils";
```

Comment:
@jrowny Sorry that I overlooked this when I reviewed #871. I didn't spot that this was a `type`-level export statement, which gets removed from the final bundle and is a TypeScript-only construct. Of course, the `toPlainLson` function needs to end up in the bundle, so I made this a normal `export` here.